src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -72,6 +72,23 @@
  * <pre>
  * &lt;module name=&quot;MissingOverride&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test extends SuperClass {
```

Comment:
From check description
`This check will log a violation if using the @inheritDoc tag on a method that is not valid (ex: private, or static method).`
Please add examples to show this

Reply:
I am done

Reply:
and update the first post~